hs_core/templates/pages/create-resource.html

```diff
@@ -1,365 +0,0 @@
-                            </div>
-                        {% endif %}
-
-                        {% if validation_error %}
-                            <div id="file-type-error" class="alert alert-danger alert-dismissible">
-                                <button type="button" class="close" data-dismiss="alert" aria-label="Close"><span
-                                        aria-hidden="true">&times;</span></button>
-                                {{ validation_error }}
-                            </div>
-                        {% endif %}
-                    </div>
-
-                    <div class="col-xs-12 col-sm-4 col-md-3">
-                        <ul id="constraints" class="list-group">
-                            <li id="required-types" class="list-group-item list-group-item-success">The following file types are required:
-                                <span>
-                                </span>
-                            </li>
-                            <li id="res-empty" class="list-group-item list-group-item-danger">This resource must have content files.</li>
-                            <li id="single-file" class="list-group-item">
-                                Single file to upload must be of type <span id="single-file-type"></span>
-                            </li>
-                            <li id="same-file-names" class="list-group-item" >Your files must share the same file name.
-                            </li>
-                        </ul>
-
-                        <ul class="list-group">
-                            <li class="list-group-item">Files you upload here will be grouped together into a
-                                "Resource" in HydroShare.
-                            </li>
-                            <li class="list-group-item">Files uploaded directly from a browser are limited to 1 GB per
-                                file.
-                            </li>
-                            <li class="list-group-item">
-                                <p>Use iRODS to add files larger than 1 GB. Any iRODS server may be used. If you do not
-                                have an account on an iRODS server you can create a HydroShare iRODS account from your
-                                 <a href="/user/{{ user.id }}/">User Profile page</a>.</p>
-
-                                <p>See <a href="https://help.hydroshare.org/creating-and-managing-resources/"
-                                          target="_blank">Creating and Managing Resources</a> for more information on
-                                    working with iRODS directly.</p>
-                            </li>
-                        </ul>
-                    </div>
-                </div>
-            </div>
-        </div>
-
-        <div class="row">
-            <div class="col-xs-12 text-right">
-                <span class="btn btn-default btn-cancel-create-resource">Cancel</span>

```

Comment:
We still have CSS and JavaScript code for this file that needs to be removed as well. 

For example, there are CSS rules for this ".btn-cancel-create-resource" button as well as some code in our scripts.

Reply:
OK. I will try to catch those css rules and javascript code that are not in use any more due to removal of this template file.

Reply:
@Maurier Have just pushed a commit to clean up unused css rules and an unused javascript function. Let me know if you notice any further needed cleanup that I could have missed.

Reply:
@hyi  I'll take a look and post anything I find in this comment thread. For now, here is one:
In https://github.com/hydroshare/hydroshare/blob/3406-remove-create-res-irods-userzone/theme/static/css/hs-file-icons.css, we can remove all selectors using '.dz-details' since this was only used in the Create Resource page.

Reply:
Similarly, all rules using "#resource-content" can be removed.

Reply:
Rules for "#btn-next-step, #btn-previous-step" can be removed as well.

Reply:
Rules for "#btn-remove-all-files" can be removed as well.

Reply:
CSS rules and JavaScript can be removed for ".hs-dropzone-wrapper"

Reply:
CSS rules can be removed for "#dropdown-resource-type".

Reply:
CSS rules can be removed for "#btn-remove-all-files"

Reply:
@hyi - I think that's all of it.

Reply:
@Maurier Thanks for the pointers. While removing all selectors using .dz-details rule, I found it is being used in dropzone.js here: https://github.com/hydroshare/hydroshare/blob/3406-remove-create-res-irods-userzone/theme/static/js/dropzone.js#L396. Are you sure it is OK to remove .dz-details rule given that it is still used?

Reply:
@hyi - Yes, this usage is not related to our CSS rule. I can confirm it can be deleted.

Reply:
@Maurier I have committed more cleanup according to your pointers as well as a few more css rules that I came across that are not used. Take a look and let me know if you found more cleanup that I can take care of.